### PR TITLE
NV 599 create cta default buttons

### PR DESCRIPTION
**Pull request by djabarovgeorge**

### CODE REVIEW

File: packages/notification-center/src/shared/config/notificationItemButton.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export enum ButtonType {
+  PRIMARY = 'primary',
+  SECONDARY = 'secondary',
+}
```
Comment (djabarovgeorge) [inline packages/notification-center/src/shared/config/notificationItemButton.tsx:4]: Remove and use Enum that added in the shared lib

File: packages/notification-center/src/components/notification-center/components/notification-item/NorificationItemButton.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  );
+}
+
+const StyledButton = styled(MantineButton)<{ styles: IButtonStyles }>`
+  color: ${({ styles }) => styles.color};
+`;
+
+export function MantineButton({ ...props }) {
+  return <Button {...props} />;
+}
```
Comment (scopsy) [inline packages/notification-center/src/components/notification-center/components/notification-item/NorificationItemButton.tsx:29]: Not sure why do you need to add this wrapper here, you could just import:
```
import { Button as MantineButton } from '@mantine/core';
```

File: packages/notification-center/src/shared/config/notificationItemButton.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+export interface IButtonStyles {
+  color: string;
+}
+
+const primaryButton: INotificationButtonStyles = {
+  key: ButtonTypeEnum.PRIMARY,
+  value: {
+    color: 'blue',
```
Comment (scopsy) [inline packages/notification-center/src/shared/config/notificationItemButton.tsx:15]: What are those? Do we actually do them blue?

Comment (djabarovgeorge) [reply packages/notification-center/src/shared/config/notificationItemButton.tsx:15]: Yea those are the reused design for the buttons

File: packages/notification-center/src/components/notification-center/components/notification-item/ActionContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import { ButtonTypeEnum, IMessageAction } from '@novu/shared';
+
+export interface IActionContainerProps {
+  actions?: IMessageAction[];
+}
+
+export function ActionContainer(props: IActionContainerProps) {
+  const buttons = props?.actions;
+
+  sortButtonsByEnum(buttons);
```
[the remark this answers is not in the document]
Comment (djabarovgeorge) [reply packages/notification-center/src/components/notification-center/components/notification-item/ActionContainer.tsx:12]: the sort is removed. so no issue there

**Pull request merged**